src/ray/gcs/gcs_server/gcs_node_manager.cc

```diff
@@ -226,13 +228,33 @@ void GcsNodeManager::HandleGetAllNodeInfo(rpc::GetAllNodeInfoRequest request,
     filter_state = request.filters().state();
   }
   std::string filter_node_name = request.filters().node_name();
-  auto filter_fn = [&filter_node_id, &filter_node_name](const rpc::GcsNodeInfo &node) {
+  std::string filter_virtual_cluster_id = request.filters().virtual_cluster_id();
```

Comment:
```suggestion
  const std::string &filter_virtual_cluster_id = request.filters().virtual_cluster_id();
```

Reply:
I think it's better to align to the upper lines